### PR TITLE
MUL-1206: ETH address prefix

### PR DESCRIPTION
**Pull request by Enmk**
Fixed Ethereum address to be always prefixed with '0x';
Added Ethereum-specific test cases that ensure that validate_address() fails;
Updated generic test cases to ensure that validate_address() fails, based on modified version of valid address.

### CODE REVIEW

File: multy_test/serialized_keys_test_base.cpp [hunk 1]
```diff
@@ -101,3 +101,25 @@ TEST_P(CheckAddressTestP, validate_address)
...
+
+    EXPECT_ERROR(validate_address(blockchain_type,
+            // remove last symbol
+            test_address.substr(0, test_address.size() - 1).c_str()));
+
+    EXPECT_ERROR(validate_address(blockchain_type,
+            // remove first symbol
+            test_address.substr(1, test_address.size() - 1).c_str()));
+
+    EXPECT_ERROR(validate_address(blockchain_type, ""));
```
Comment (PashaKlybik) [inline multy_test/serialized_keys_test_base.cpp:124]: You check it in ``` multy_test/test_ethereum_account.cpp``` on ```EthereumBlockchainTest, validate_address``` case.

**Review by PashaKlybik — COMMENTED** [review]
Fix please.

**Comment by Enmk** [comment]
Fixed

**Review by PashaKlybik — APPROVED** [review]
Ok!

**Pull request merged**